app/Http/Controllers/GroupController.php

```diff
@@ -23,6 +25,34 @@ public function __construct()
         $this->beforeFilter('auth', array('on' => array('post', 'put', 'delete')));
     }
 
+    /**
+     * Store newly created Group.
+     *
+     * @return Response
+     */
+    public function store(StoreGroupRequest $request)
```

Comment:
Also, I'm wondering why this doesn't throw an error.  By default, this should receive an instance of `Request`, not `StoreGroupRequest`, and should throw a type error?  Where is this being cast into the `StoreGroupRequest` type?